index-debug.html

```diff
@@ -82,7 +82,14 @@
+
+        w2utils.lock(loadingDiv, myHTML, true);
```

Comment:
Do we see the spinner with that logo ? It is set to « true » so you have the spinner on the left and the logo on the right ?